lago_python_client/models/fee.py

```diff
@@ -57,6 +57,7 @@ class FeeResponse(BaseResponseModel):
     taxes_precise_amount: Optional[str]
     total_amount_currency: Optional[str]
     units: Optional[float]
+    total_aggregated_units: Optional[float]
```

Comment:
Shouldn't it be a `str` as described in https://github.com/getlago/lago-openapi/pull/459?

Reply:
I commented this case in https://github.com/getlago/lago-openapi/pull/459
`units` and `total_aggregated_units` is the same data type, so we should change `units`, too.